src/data/defaults.txt

```diff
@@ -17,6 +17,7 @@ global	chatBeep	false
 global	chatFontSize
 global	chatLinksUseRelay	false
 global	chatStyle	0
+global	checkRssUpdate	false
```

Comment:
Do you also need to set a default value for lastRssUpdate?

I'm worried that Long.parseLong("") will throw an exception.

Reply:
It already does, it's an existing preference and it defaults to 0